release-notes/aws/v9.0.2.md

```diff
@@ -0,0 +1,18 @@
+## :zap: Giant Swarm Release 9.0.2 for AWS :zap:
+
```

Comment:
```suggestion

**If you are upgrading from 9.0.1, upgrading to this release will not roll your nodes. It will only update the apps.**

```

Reply:
@sslavic , this applies here too right, because nginx version is defined here and not inside cluster operator?

Reply:
Well spotted, thank. Suggestion has been applied. Please take another look.